mParticle-Apptimize/MPKitApptimize.m

```diff
@@ -130,6 +133,52 @@ - (nullable NSString*) configValueForKey:(NSString*)key {
     return value;
 }
 
+- (void) configureExperimentTracking {
+    BOOL enable = [self configValueForKey:TRACK_EXPERIMENTS];
+    if (enable) {
+        [[NSNotificationCenter defaultCenter] addObserver:self
+                                                 selector:@selector(experimentDidGetViewed:)
+                                                     name:ApptimizeTestRunNotification
+                                                   object:nil];
+    }
+}
+
+- (void) experimentDidGetViewed:(NSNotification*)notification {
+    if (![notification.userInfo[ApptimizeTestFirstRunUserInfoKey] boolValue]) {
+        return;
+    }
+
+    NSMutableArray *profileAttributeStrings = [NSMutableArray new];
+
+    for (id<ApptimizeTestInfo> test in [[Apptimize testInfo] allValues]) {
+        if (test.userHasParticipated) {
+            // We only want experiments the user has actually participated in
+            [profileAttributeStrings addObject:[NSString stringWithFormat:@"%@-%@",test.testName,test.enrolledVariantName]];
+        }
+    }
+
+    [[MParticle sharedInstance] setUserAttribute:@"Apptimize experiment" values:profileAttributeStrings];
+
+    // Apptimize doesn't notify with IDs, so we iterate over all experiments to find the matching one.
+    NSString *name = notification.userInfo[ApptimizeTestNameUserInfoKey];
+    NSString *variant = notification.userInfo[ApptimizeVariantNameUserInfoKey];
+    [[Apptimize testInfo] enumerateKeysAndObjectsUsingBlock:^(id key, id<ApptimizeTestInfo> experiment, BOOL *stop) {
+        BOOL match = [experiment.testName isEqualToString:name] && [experiment.enrolledVariantName isEqualToString:variant];
+        if (!match) {
+            return;
+        }
+        MPEvent *event = [[MPEvent alloc]initWithName:@"Apptimize experiment"
```

Comment:
Similar as my comment in the Android PR - do you want to exclude this event by name in the logEvent implementation of the kit, so it doesn't go into Apptimize?